.github/workflows/build-deb.yaml

```diff
@@ -123,6 +133,22 @@ jobs:
        github.event_name == 'release' }}
 
     steps:
+    # FIXME: Use dynamic outputs when possible: https://github.com/actions/runner/pull/2477
+    - name: Setup job variables
+      run: |
+        set -exuo pipefail
+
+        json_output='${{ toJSON(needs.build-deb-package.outputs) }}'
+        outputs=(
+          pkg-src-changes
+        )
+
+        for var in "${outputs[@]}"; do
+          versioned_var="${var}-${{ matrix.ubuntu-version }}"
+          v=$(echo "${json_output}" | jq -r '."'"${versioned_var}"'"')
```

Comment:
what's going on with all these quotes? if we want that to evaluate to `."value"`, could we use the more readable `".\"${versioned_var}\""` instead?

Reply:
Yeah, I went old school :)